actions/campaigns.go

```diff
@@ -65,6 +66,16 @@ func StartCampaign(c *gin.Context) {
 		return
 	}
 
+	sc, err := storage.GetScheduledCampaign(c, campaign.ID)
```

Comment:
We should add a `Schedule` entity to the campaign so we can preload it, instead of fetching it like this.

Reply:
@gudgl just pull from shared/ it's already done.